build.gradle

```diff
@@ -49,7 +49,9 @@ dependencies {
                     "com.sun.jersey:jersey-core:1.19",
                     "com.sun.jersey.contribs:jersey-multipart:1.19",
                     "org.apache.taglibs:taglibs-standard-impl:1.2.5",
-                    "org.jsoup:jsoup:1.9.2"
+                    "org.jsoup:jsoup:1.9.2",
+                    "org.apache.poi:poi:3.9",
```

Comment:
The reason was the [usage](https://mvnrepository.com/artifact/org.apache.poi/poi) of versions. This version had the highest current usage up to now. Shall I change to the latest ?